components/x-audio/src/components/TimeRemaining.jsx

```diff
@@ -0,0 +1,30 @@
+import { h } from '@financial-times/x-engine';
+import * as PropTypes from 'prop-types';
+import classNameMap from './classnames-helper';
+import formatToHHMMSS from './format-seconds-to-hhmmss';
+
+const formatToMinsRemaining = (targetSeconds) => {
+	const minutes = Math.floor((targetSeconds / 60))
```

Comment:
Just thinking about `Math.floor` here... 

I guess for the last 60 seconds it will always say "0 minutes remaining", which could feel like a long time? I.e. when I see "0 minutes remaining" I would expect it to end very soon.

Would `Math.round` work better? So it would only flip to 0 when less than 30 seconds?

Reply:
I will tackle this by another PR after asking designer Luke whether it need to be change into seconds when the remaining is less than minute or not.